### DISCUSSION TITLE
Unable to connect with my Coinbase wallet anymore

### DISCUSSION THREAD

**Discussion by Punk1028** [WalletConnect/walletconnect-monorepo#3522] [Developer Support] [answered]
Hi there, I was able to connect until now on the https://app.ipor.io with my Coinbase wallet. Unfortunately, I am unable to connect anymore. What can I do?  Thanks for the help

**Comment by glitch-txs** [comment, marked as answer]
Hey, this is a bit unrelated to WalletConnect, Coinbase uses a different protocol and SDK, so it's up to the website to add support for it.
A workaround I could suggest is opening the Coinbase wallet's browser on your phone, go to the website there and click on MetaMask button to connect.

If you're using Coinbase on desktop as an extension, then I'd suggest disabling all wallets and using the MetaMask button 👍
